docs/contributing/event_organizer/event_organizer.rst

```diff
@@ -0,0 +1,65 @@
+Event organizer
+###############
+
+.. vale off
+
+Organizing Mautic events
+************************
+
+Events are the beating heart of a thriving open source community, and we have quite a lot of them happening throughout the year. We hold an in-person event annually which moves continent each year, in addition to Mautic Conference Global which is held virtually. Local communities can also host a MautiCamp event - a localised conference for people in that region - and Mautic Meetups which are smaller monthly gatherings of Mauticians.
+
+Whether you want to help with the big international events or the regional and local events, there are lots of opportunities to put your expertise to good use and help bring Mauticians together in-person, online, or in a combination of the two.
+
+.. _Getting started event organizer:
+
+Getting started
+***************
+
+Events fall under the :doc:`/teams/community_team/community_team`, so your first step is to join the Community Team Slack channel. Get an invite to join :xref:`Mautic Community Slack` if you aren't there already, and then you can join the ``#t-community`` channel.
+
+The team meets fortnightly on Slack, so you can scroll back and check what was being discussed in previous meetings.
+
+Finding open tasks
+==================
+
+Each team maintains its own Jira board, where you can find the current open tasks and a backlog of things that the team plans to work on.
+
+.. Add link to the Community Team's Jira board. It's in "jira_community_team_board.py" at the "links" folder
+
+Feel free to access the Community Team's [Jira board].
```

Comment:
Same here, let's leave it unlinked if you're intending to add the link in a separate PR, but I'm not sure why you wouldn't include it in this one?

Reply:
I've included the link here.